tmux-cssh

```diff
@@ -53,23 +57,36 @@ if [ -z "${hosts}" ]; then
 fi
 
 # Find a name for a new session
-n=0; while tmux has-session -t "${tmux_session_name}-${n}" 2>/dev/null; do n=$(($n + 1)); done
-tmux_session="${tmux_session_name}-${n}"
+n=0; while tmux has-session -t "${tmux_name}-${n}" 2>/dev/null; do n=$(($n + 1)); done
+tmux_session="${tmux_name}-${n}"
```

Comment:
the window naming that was added in the recent commit works. one question though: why is it adding number suffix add the end? why not just use the given name?

Reply:
is to be consistent with session naming
https://github.com/peikk0/tmux-cssh/blob/dffe8ee34649b398ac3b9e2b12908f1d74e1cd96/tmux-cssh#L56-L57

Reply:
ok then :)